src/d/actor/d_a_obj_tntrap.cpp

```diff
@@ -4,172 +4,643 @@
 //
 
 #include "d/actor/d_a_obj_tntrap.h"
+#include "d/actor/d_a_player.h"
+#include "d/actor/d_a_ship.h"
+#include "d/d_com_inf_game.h"
 #include "d/d_procname.h"
 #include "d/d_priority.h"
+#include "m_Do/m_Do_mtx.h"
+
+
+const char l_arcname[] = "TnTrap";
```

Comment:
These variables should be inside an anonymous namespace based on the original symbol names.